bezier-rs/docs/interactive-docs/src/utils/drawing.ts

```diff
@@ -18,18 +18,18 @@ export const drawLine = (ctx: CanvasRenderingContext2D, p1: Point, p2: Point): v
 	ctx.stroke();
 };
 
-export const drawPoint = (ctx: CanvasRenderingContext2D, p: Point, stroke = "black"): void => {
+export const drawPoint = (ctx: CanvasRenderingContext2D, p: Point, r: number, stroke = "black"): void => {
```

Comment:
Please use full word variable names, especially when they are part of a function signature. So `point` instead of `p` and `radius` instead of `r` (assuming that even means radius).